jenkins-pipelines/performance/branch-perf-v15/scylla-master/perf-regression/scylla-master-perf-regression-throughput-non-shard-aware-i4i-tablets.jenkinsfile

```diff
@@ -0,0 +1,14 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+perfRegressionParallelPipeline(
+    backend: "aws",
+    region: "us-east-1",
+    test_name: "performance_regression_test.PerformanceRegressionTest",
+    test_config: '''["test-cases/performance/perf-regression.100threads.30M-keys-i4i.yaml", "configurations/disable_shared_aware_drivers.yaml", "configurations/disable_speculative_retry.yaml"]''',
+    sub_tests: ["test_write", "test_read", "test_mixed"],
+    test_email_title: "throughput - non-shard-aware / tablets",
```

Comment:
non-shard-aware / tablets - we need to think if we need such a case - IMHO this case should be tested with tablets aware driver.

Reply:
This actually a very interesting question 

If we disable shared aware like here, is tablet aware still kicks in ?

Reply:
after discussing with @roydahan 
we removed the non-shard-aware from the weekly runs

we could run it if need, but from now on, only shard-aware driver would be used